gatsby/src/components/accordion/accordion-section.tsx

```diff
@@ -26,6 +26,7 @@ const AccordionSection: React.FC<Section> = ({ title, text }) => {
           'justify-content-between',
           'align-items-center',
         )}
+        onClick={toggleSection}
```

Comment:
`onClick` na generic elemente. Stacilo by asi pridat `role="button"`

Reply:
@Haaxor1689 nerozumím jak to myslíš? Potřebuji aby ta sekce šla otevřít i při kliknutí na nadpis

Reply:
Ze `div` by nemal mat `onClick` koli accesibility. Alebo po zmenit na `button` a odstranit styly defaultne.

Reply:
@Haaxor1689 souhlas, zas ten button tam zanořený je, teoreticky by nemusel být problém. Zatím nechme, kdyžtak se fixne až se bude řešit přístupnost celkově